client/src/pages/AdminLogin/apiCalls.js

```diff
@@ -0,0 +1,15 @@
+const loginAdmin = (username, password) => {
+  // TODO stubbed api call for now
+  // eslint-disable-next-line no-unused-vars
+  const params = { username, password };
+
+  return new Promise((resolve) => {
+    resolve({
```

Comment:
is this the stubbed out api call that will return a token later? seems confusing for now but if that's the case then it makes sense